content/SCALE/SCALECLIReference/Service/_index.md

```diff
@@ -24,18 +24,17 @@ You can enter commands from the main CLI prompt or from the **service** namespac
 The `get_instance` command displays the id, name, and status of a service. 
 
 {{< expand "Checking the Status of a Service" "v" >}}
-The `get_instance` command requires you to include the `id` option in the command string. After entering the command correctly, it returns a table with multiple outputs.
+#### Description
+The `get_instance` command requires you to include the `id` property in the command string. 
+Enter the command, then press <kbd>Enter</kbd>.
+After entering the command correctly, it returns a table with multiple outputs.
 
+#### Usage
 From the CLI prompt, enter:
 
 <code>service get_instance id=<i>number</i></code>
 
-From the **service** prompt, enter:
-
-<code>get_instance id=<i>number</i></code>
-
-Where:
-* `number` is the service id. For example, if the ssh service id is 11, enter 11.
+Where *number* is the service id. For example, if the ssh service id is 11, enter 11.
```

Comment:
It's not exactly the point of this PR, but it might be a good idea to include something about using `query` to return a list including service id numbers. That or developing the query section a bit more to include what types of data it returns since some commands require service id, some name, some pid.